### DISCUSSION TITLE
Filter on collection for tables?

### DISCUSSION THREAD

**Discussion by NicolasSacC** [filamentphp/filament#4432] [Help]
Hi!

I have a custom attribute that gives me the subscription status of an organization based on a few parameters on the users table. I tried to create a filter, but it doesn't work since it queries the DB.

Is there a way to ask the filter to work on the collection instead, using the computed attribute instead of looking in the database?

I know this isn't optimal for performances, but it's quite useful in some use cases.

Thanks!

**Comment by danharrin** [comment]
The only way would  be through https://github.com/calebporzio/sushi
